app/Providers/AppServiceProvider.php

```diff
@@ -29,6 +29,18 @@ public function boot()
         Validator::extend('student_number', function ($attribute, $value, $parameters, $validator) {
             return preg_match('/^(a|pg)[0-9]+$/i', $value) === 1;
         });
+
+        Validator::extend('greater_or_equal_than_field', function ($attribute, $value, $parameters, $validator) {
```

Comment:
This is how this validation is implemented in **laravel 5.7**
Maybe you should take a look to make this method similar, just to guarantee we are not missing any case.

`https://github.com/laravel/framework/blob/ac745730492ef23a04ce614228d11e496feb625d/src/Illuminate/Validation/Concerns/ValidatesAttributes.php#L879`